asciidoc/testing/dotnet_testing_selenium.adoc

```diff
@@ -219,30 +219,6 @@ One of the best methods to test your locator is working is by trying it in the b
 
 link:https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes[Data attributes] can help identify the elements under test.
 
-=== Example
```

Comment:
As per other comment, I think this should be retained as it is not cypress code, it is C# Selenium

Reply:
Restored